app/models/ability.rb

```diff
@@ -48,14 +48,15 @@ def initialize(user)
       can [:create], AgreementLetter
       can [:new, :create], Request
       cannot :view_personal_details, ApplicationLetter, user: { id: !user.id }
+      can [:show, :index, :archive], Event
```

Comment:
@hendraet told me that it should be possible for guests to see events, so we thought to skip the authorization of these methods.